### ISSUE TITLE
Unused history check

### ISSUE THREAD

**Issue by aocenas** [yahoo/fluxible-router#106]
Hi, just noticed that this check: https://github.com/yahoo/fluxible-router/blob/master/lib/handleHistory.js#L41, is never changed to true. So I assume, that what ever it should prevent (creating more history listeners I think), it does not do that right now. Did not really check with any code samples, but simple search for that variable never turns any assignment which would change it to true.

**Comment by mridgway** [comment]
Good catch. That was put in place so that there aren't multiple `handleHistory` higher-order components on the page at once... but it seems somewhere along the line it was broken. I'll take a look.
